public/wp-content/themes/newsup/inc/ansar/hooks/hooks.php

```diff
@@ -1,19 +1,19 @@
-<?php 
+<?php
 /**
 PHP functions & Hooks:
-Theme URL: https://wordpress.org/themes/newsphere/, 
+Theme URL: https://wordpress.org/themes/newsphere/,
 http://afthemes.com/, (C) 2020 AF Themes, GPLv2
 */
-if (!function_exists('newsup_banner_trending_posts')):
+if (!function_exists('newsup_banner_exclusive_posts')):

```

Comment:
The parent theme was checking for an outdated hook name. This update allows it to check that the child has now replugged the function it would have otherwise defined itself.